packages/driver/src/rawConn.ts

```diff
@@ -53,12 +53,15 @@ function getTlsOptions(config: ResolvedConnectConfig): tls.ConnectionOptions {
 
   const tlsOptions: tls.ConnectionOptions = {
     ALPNProtocols: ["edgedb-binary"],
-    // XXX Deno doesn't support this and that means it won't
-    // work with EdgeDB Cloud.
-    servername: config.address[0],
     rejectUnauthorized: tlsSecurity !== "insecure",
   };
 
+  if (net.isIP(config.address[0]) !== 0) {
```

Comment:
If it is an IP, `isIP` will return 4 or 6, so this condition will be true, so we will set the `servername`.

I think this should be backwards. We should set the `servername` when this is not an IP.

Reply:
Oh, right 🤦 . I'll make a boolean variable so this is more obvious.